provider/cluster_rosa_classic_resource.go

```diff
@@ -380,6 +380,20 @@ func createClassicClusterObject(ctx context.Context,
 	state *ClusterRosaClassicState, logger logging.Logger, diags diag.Diagnostics) (*cmv1.Cluster, error) {
 
 	builder := cmv1.NewCluster()
+	clusterName := state.Name.Value
+	if len(clusterName) > 15 {
```

Comment:
NP
We can extract the value into a constant `maxClusterNameLength` if we need to reuse it in other places.

Reply:
included maxClusterNameLength as constant.